DeusEx/Classes/DXRBacktracking.uc

```diff
@@ -0,0 +1,59 @@
+class DXRBacktracking extends DXRActorsBase;
+
+function FirstEntry()
+{
+    local Teleporter t;
+    local DynamicTeleporter dt;
+    local BlockPlayer bp;
+    Super.FirstEntry();
+
+    switch(dxr.localURL) {
+        case "10_PARIS_METRO":
+            foreach AllActors(class'BlockPlayer', bp) {
+                if( bp.Name == 'BlockPlayer0' ) {
+                    bp.bBlockPlayers=false;
+                }
+            }
+            dt = Spawn(class'DynamicTeleporter',,'sewers_backtrack',vect(1599.971558, -4694.342773, 13.399302));
+            dt.URL = "10_PARIS_CATACOMBS_TUNNELS#?toname=AmbientSound10";
+            dt.Radius = 160;
+            class'DXRFixup'.static._AddSwitch(Self, vect(1602.826904, -4318.841309, -250.365067), rot(0, 16384, 0), 'sewers_backtrack');
```

Comment:
not sure if I should force the player to climb back up (maybe provide a crate or 2), or just let them hit the switch